.circleci/config.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           command: cargo generate-lockfile
       - restore_cache:
           keys:
-            - v4-cargo-cache-{{ arch }}-{{ checksum "Cargo.lock" }}
+            - cargo-cache-{{ .Environment.CACHE_VERSION }}-{{ arch }}-{{ checksum "Cargo.lock" }}
```

Comment:
Easier cache management in cases when we need to invalidate earlier caches if `cargo.lock` is not changed.